lib/std/os/windows.zig

```diff
@@ -2343,6 +2360,13 @@ pub const NamespacePrefix = enum {
     fake_verbatim,
     /// `\??\`
     nt,
+
+    pub fn prefixLen(namespace_prefix: NamespacePrefix) i32 {
+        switch (namespace_prefix) {
+            .none => 0,
+            .local_device, .verbatim, .fake_verbatim, .nt => 4,
+        }
+    }
```

Comment:
Personally I'd go with
```suggestion
    /// The length of all namespace prefixes (except `none`)
    pub const len = 4;
```
AFAIK it's unlikely for this to change or need to vary.

Usage would then look like:

```zig
const unprefixed_path = final_path[NamespacePrefix.len..];
```

Reply:
sure thing! will get that in a mo.